src/components/Common/ButtonV2.tsx

```diff
@@ -101,7 +101,7 @@ export const buttonStyles = ({
   shadow = !ghost,
 }: ButtonStyleProps) => {
   return classNames(
-    "inline-flex h-min cursor-pointer items-center justify-center gap-2 whitespace-pre font-medium outline-offset-1 transition-all duration-200 ease-in-out disabled:cursor-not-allowed disabled:bg-secondary-200 disabled:text-secondary-500",
+    "inline-flex h-min cursor-pointer items-center justify-center gap-2 whitespace-pre font-medium outline-offset-1 transition-all duration-200 ease-in-out disabled:cursor-not-allowed disabled:bg-secondary-200 disabled:text-secondary-500 mtrequired",
```

Comment:
As I mentioned in the previous comment, let's not change anything in ButtonV2, but adjust where the issue occurs.